plugins/azure_sentinel/icon_azure_sentinel/util/api.py

```diff
@@ -0,0 +1,276 @@
+import json
+import logging
```

Comment:
There is a validator error connected with this line.
`Validator "LoggingValidator" failed! 
	Cause: One or more files imports logging, update to self.logger`
```suggestion
from logging import ...
```